batch_import/README.md

```diff
@@ -244,88 +243,134 @@ Which will output the following:
 
-    Here is an example of how to run the script to download all labels for
-    the `job_id` SjksdkjoowlkjlSDFifajoijerWE231dsdf:
+    Here is an example of how to run the script to recover a dictionary of lables for the job with job_id=SjksdkjoowlkjlSDFiwjoijerSDRdsdf.
+    This will obtain the bookmark of labels for the job and will write this output to the file /tmp/job_labels.json
+
+    python download_labels.py --output_file /tmp/job_labels.json SjksdkjoowlkjlSDFiwjoijerSDRdsdf
 
-    python download_labels.py -v SjksdkjoowlkjlSDFifajoijerWE231dsdf /home/me/outputfile.json
+    If you just want to list all jobs that belong to your account, you can do the followng
+
+    python download_labels.py --output_file /tmp/job_list.json
+
+    which will write the list of jobs that belong to the user to the file '/tmp/job_list.json'
 
 ```
 
-This script accepts a job_id, queries the [Camio API](https://api.camio.com/#jobs) to get the job definition, then uses 
+This script accepts a `job_id`, queries the [Camio API](https://api.camio.com/#jobs) to get the job definition, then uses 
 the job definition and the [Camio search API](https://api.camio.com/#search) to go through all events that belong to that job. While
 going through all of these events, it assembles the labels into a json object and writes this object to the output file (which can be specified 
 by you or simply defaults to `{{job_id}}_results.json`). This json object has the following structure
 
 ```json
 {
-    "earliest_date": "1970-01-01T12:00:00.0000-0000",
-    "latest_date": "1970-01-01T12:30:00.0000-0000",
-    "job_id": "sdfslkjfjowejoijsldkjflksjdf",
-    "labels": {
-         "1970-01-01T12:00:01.0345-0000": {
-              "cameras": {
-                  "camera_name_1": {
-                      "name": "camera_name_1",
-                      "camera_id": "wer9ikjsodfj09j34rojiosj"
-                   }
-                },
-                "labels": [
-                    "dog",
-                    "car",
-                    "human"
-                ]
-            }
-         },
-         "1970-01-01T12:00:03.02089-0000": {
-              "cameras": {
-                  "camera_name_1": {
-                      "name": "camera_name_1",
-                      "camera_id": "wer9ikjsodfj09j34rojiosj"
-                   }
-                },
-                "labels": [
-                    "human",
-                    "suv",
-                    "tree"
-                ]
-            }
-        }
+  "job_id": "ag1zfmNhbWlvbG9nZ2VychALEgNKb2IYgIDI15PVuwgM",
+  "latest_date": "2016-10-09T06:22:36.993000",
+  "earliest_date": "2016-10-09T05:02:37.000"
+  "labels": {
+    "2016-10-09T05:10:31.456-0000": {
+      "camera": {
+        "name": "C2_Hi"
+      },
+      "labels": [
+        "bass",
+        "_color_white",
+        "47390561bf685356557fc083c410914dc4b6fde6",
+        "bluefin",
+        "human",
+        "salmon",
+        "_color_green",
+        "yellowfin",
```

Comment:
@jhallard the labler needs to use "yellowfin tuna", "bluefin tuna" to match your whitelist, and remove "bass" and "trout" just to keep in salt water family (even though fake demo)